_health-care/_js/_form.js

```diff
@@ -1,21 +1,47 @@
 var HealthApp = (function() {
   "use strict";
 
-  var identityTransform = function(formData, arg) { return arg; };
-  var extractTransform = function(formData, arg) { return formData[arg]; };
+  function identityTransform(formData, arg) { return arg; }
+  function extractTransform(formData, arg) { return formData[arg]; }
```

Comment:
Oh interesting, so since you scoped all of this within `var HealthApp = function() {}`, function declarations technically aren't allowed? Learning new things!